models/api.models.js

```diff
@@ -113,3 +113,14 @@ exports.deleteCommentById = (comment_id) => {
     }
   });
 };
+
+exports.fetchAllUsers = () => {
+  const queryStr = `
+    SELECT username, name, avatar_url
```

Comment:
are there other fields that you're trying to avoid? if not then you can select everything here :)